libnavigation-ui/src/main/java/com/mapbox/navigation/ui/route/MapRouteProgressChangeListener.kt

```diff
@@ -59,19 +59,18 @@ internal class MapRouteProgressChangeListener(
     private fun updateRoute(directionsRoute: DirectionsRoute?, routeProgress: RouteProgress) {
         val currentRoute = routeProgress.route()
         val hasGeometry = currentRoute?.geometry()?.isNotEmpty() ?: false
-        if (currentRoute != null && hasGeometry && currentRoute != directionsRoute) {
-            routeLine.draw(currentRoute)
-            routeArrow.addUpcomingManeuverArrow(routeProgress)
+        if (hasGeometry && currentRoute != directionsRoute) {
+            routeLine.draw(currentRoute!!)
```

Comment:
Is this safe? 🤔

Reply:
hasGeometry can only be true if currentRoute is not null.